example/src/component/ProgressBar.tsx

```diff
@@ -17,11 +17,6 @@ export class ProgressBar extends React.Component<Props> {
     const clampedSize = Math.max(0, Math.min(1, size));
     return (
       <View
-        accessibilityValue={{
```

Comment:
Why it has been removed ?

Reply:
its breaking the app as for out of bounds violation